apps/trading/client-pages/market/market-selector-item.tsx

```diff
@@ -106,25 +103,34 @@ const MarketData = ({ market }: { market: MarketMaybeWithDataAndCandles }) => {
           </p>
         )}
       </div>
-      <DataRow value={volume} label={t('24h vol')} />
+      <DataRow
+        value={
+          <Last24hVolume
```

Comment:
suggestion (blocking): Lets not re-use this component as the design is different. Instead refactor so that the biz logic can be shared by a hook that the header-stat and market-selector-item can use to retrieve the formatted data but render different UI